skbio/sequence/_nucleotide_mixin.py

```diff
@@ -14,24 +14,16 @@
 import numpy as np
 
 from skbio.util import classproperty
-from ._iupac_sequence import IUPACSequence, _motifs as parent_motifs
+from ._iupac_sequence import _motifs as parent_motifs
 
 
-class NucleotideSequence(with_metaclass(ABCMeta, IUPACSequence)):
-    """Abstract base class for storing an IUPAC nucleotide sequence.
+class NucleotideMixin(with_metaclass(ABCMeta, object)):
```

Comment:
Yes, but I don't think it needs to be abstract, it's a mixin, I've never heard of anyone trying to instantiate a mixin and expecting anything to happen.